packages/ui/v2/core/layouts/SettingsLayout.tsx

```diff
@@ -340,18 +358,21 @@ export default function SettingsLayout({
       withoutSeo={true}
       flexChildrenContainer
       {...rest}
-      SidebarContainer={<SettingsSidebarContainer className="hidden lg:flex" />}
+      SidebarContainer={
+        <>
+          {/* Mobile backdrop */}
+          {sideContainerOpen && (
+            <button
+              onClick={() => setSideContainerOpen(false)}
+              className="fixed top-0 left-0 z-10 h-full w-full bg-black/50">
+              <span className="sr-only">{t("hide_navigation")}</span>
```

Comment:
do we wanna translate sr-only strings, too?

Reply:
Yes